tgui/packages/tgui/interfaces/BigManipulator.tsx

```diff
@@ -33,6 +32,82 @@ type PrioritySettings = {
   priority_width: number;
 };
 
+const DelayControls = ({ act, data }) => {
+  const { delay_step, delay_value, min_delay, max_delay } = data;
+  return (
+    <Stack>
+      <Stack.Item style={{ marginRight: '10px' }}>Delay:</Stack.Item>
+      <Stack style={{ width: '100%' }}>
+        <Stack.Item>
+          <Button
+            icon="backward-step"
+            onClick={() =>
+              act('changeDelay', {
+                new_delay: min_delay,
+              })
+            }
+          />
+        </Stack.Item>
+        <Stack.Item grow>
+          <Slider
+            style={{ marginTop: '-5px' }}
+            step={delay_step}
+            my={1}
+            value={delay_value}
+            minValue={min_delay}
+            maxValue={max_delay}
+            unit="sec."
+            onDrag={(e, value) =>
+              act('changeDelay', {
+                new_delay: value,
+              })
+            }
+          />
+        </Stack.Item>
+        <Stack.Item>
+          <Button
+            icon="forward-step"
+            onClick={() =>
+              act('changeDelay', {
+                new_delay: max_delay,
+              })
+            }
+          />
+        </Stack.Item>
+      </Stack>
+    </Stack>
+  );
+};
+
+const ConfigRow = ({ label, content, onClick, tooltip, selected = false }) => (
```

Comment:
I know we can technically do this and it's valid JS, but it's also breaking convention to not include the return

In writing it this way it's losing its type safety. Each of these props are untyped. If you inline the type, it becomes a redundant mess. Please,

```tsx
type ConfigRowProps = {
// prop types
};

const ConfigRow = (props: ConfigRowProps) => {
const { // your props } = props:

return (
```